content/docs/user-guide/settings/az-console-how-to-use-settings-registry.md

```diff
@@ -0,0 +1,173 @@
+---
+description: Describes how to execute AZ Console Commands using the Settings Registry
+title: Using AZ::Conosle to access the Settings Registry
+linktitle: Using AZ::Conosle to access the Settings Registry
+weight: 400
+---
+
+Preamble
+========
+
+The AZ::Console has been extended with support for the Settings Registry to allow invoking console commands via setting a JSON value underneath a specific key in the Settings Registry.
+The AZ::Console hooks into the Settings Registry Notification System as well as the JSON Patching Reporting system in order to detect changes to Settings Registry keys underneath the JSON object of "/Amazon/AzCore/Runtime/ConsoleCommands" and "/O3DE/Autoexec/ConsoleCommands"
+Where to place a console command depends on whether the user desires the console command should be merged to the bootstrap.game.\<config>.\<platform>.setreg file and therefore in the GameLauncher/ServerLauncher
+
+
+
+SettingsRegistry ConsoleCommand key differences
+
+`/Amazon/AzCore/Runtime/ConsoleCommands` - ConsoleCommands underneath this object is NOT merged to the bootstrap.game.\<config>.\<platform>.setreg
+
+`/O3DE/Autoexec/ConsoleCommands` - ConsoleCommands underneath this object is merged into the bootstrap.game.\<config>.\<platform>.setreg and therefore will execute in the GameLauncher/ServerLauncher
+
+
+
+Running Console Commands from a file
```

Comment:
Should this section mention about when these commands are run in the console lifecycle?

Reply:
Yeah I can add an explanation about deferred console commands and when they run.